src/graphics/graphics-device.js

```diff
@@ -13,6 +13,9 @@ import { Debug } from '../core/debug.js';
 const EVENT_RESIZE = 'resizecanvas';
 
 /** @typedef {import('./render-target.js').RenderTarget} RenderTarget */
+/** @typedef {import('./texture.js').Texture} Texture */
+/** @typedef {import('./render-target.js').RenderTarget} RenderTarget */
```

Comment:
You have `RenderTarget` twice. Also, I would put these in alphabetical order, personally.

Reply:
done, thanks